bot.ml

```diff
@@ -839,7 +839,18 @@ let callback _conn req body =
                         ~number:pr.issue.number
                       >>= function
                       | Ok reviews_info -> (
-                          if not (String.equal reviews_info.baseRef "master")
+                          if
+                            List.exists reviews_info.last_comments ~f:(fun c ->
+                                String.equal comment_info.id c.id
+                                && c.created_by_email)
+                          then
+                            GitHub_mutations.post_comment ~bot_info
+                              ~message:
+                                (f "@%s: This PR cannot be merged by email."
```

Comment:
I would explain the reason behind this. Something like:

```suggestion
                                (f "@%s: Merge requests sent over e-mail are not \
                                    accepted because this put less guarantee on the \
                                    authenticity of the author of the request."
```